apps/client/src/widgets/community/components/live-popular-feed/live-popular-feed.tsx

```diff
@@ -30,25 +44,23 @@ const LivePopularFeed = () => {
         onSlideEnd={() => setCurrentPage(2)}
         className={styles.carousel}
       >
-        {MOCK_FEED_CARD.map(
-          ({ id, title, content, commentCount, likeCount }) => (
-            <Carousel.Item key={id} className={styles.carouselItem}>
+        {popularFeedData?.data?.posts?.map(
+          ({ title, content, commentCount, likeCount, postId }, index) => (
+            <Carousel.Item key={index} className={styles.carouselItem}>
```

Comment:
```suggestion
          ({ title, content, commentCount, likeCount, postId }) => (
            <Carousel.Item key={postId} className={styles.carouselItem}>
```

postId로 Map을 돌리는게 좋지 않을까 싶어요! 혜린님의 의견은욥?

Reply:
좋은 것 같아요 ! 반영하겠습니다